src/App.js

```diff
@@ -45,6 +47,8 @@ function App() {
         flightTime: flightTime,
         aircraft: aircraft,
         numberOfPassengers: numberOfPassengers,
+        departureLocation: departureLocation,
+        arrivalLocation: arrivalLocation,
```

Comment:
You could use the shorthand here, because `key` and `value` are the same. More info [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer?retiredLocale=de#property_definitions) or [here](https://attacomsian.com/blog/javascript-object-property-shorthand)
```suggestion
        flightTime,
        aircraft,
        numberOfPassengers,
        departureLocation,
        arrivalLocation,
```